.github/workflows/build.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: Build uarch-based interpreter
         run: make uarch-with-toolchain
 
+      - name: Build deb package
+        run: make debian DESTDIR=$PWD/_install/usr PREFIX=/usr DEB_FILENAME=machine-emulator.deb
```

Comment:
I think you should define `MACHINE_EMULATOR_VERSION` version here, at least on tag releases to the deb package is built correctly.